notification/package.json

```diff
@@ -72,6 +72,7 @@
     "lint-staged": "^10.5.3",
     "mocha": "^8.2.1",
     "nyc": "^15.1.0",
+    "p-map": "^4.0.0",
```

Comment:
how was it working without this?

Reply:
Good question, actually I don't know 😕  It's used in the tests, I would expect them to fail.

Reply:
Looks like merge conflicts issues because its set on the master branch: https://github.com/commercetools/commercetools-adyen-integration/blob/master/notification/package.json#L55